WorkloadManagementSystem/Agent/JobAgent.py

```diff
@@ -606,6 +613,28 @@ def __rescheduleFailedJob( self, jobID, message, stop = True ):
     return self.__finish( 'Job Rescheduled', stop )
 
   #############################################################################
+  def __getCores( self ):
+    """
+    Return number of cores from gConfig and a boolean indicating corresponding to WholeNode option
+    """
+    tag = gConfig.getValue( '/Resources/Computing/CEDefaults/Tag', None )
```

Comment:
I don't understand why this location should be used. This is a location for "defaults", so, unless I don't understand what's the goal, I don't think this is a correct location for these types of Tags, that IMHO are instead specific of the single CEs.